CHANGELOG.md

```diff
@@ -12,6 +12,17 @@ Fixes:
 ------
 - ...
 
+0.1.0 (2020-02-19)
```

Comment:
should be 2021 !

Reply:
Oups hahah >_< Same error in the other PR.